app/src/main/java/com/pajato/android/gamechat/authentication/CredentialsManager.java

```diff
@@ -117,6 +128,17 @@ public void persist(@NonNull final IdpResponse response) {
         persist(new Credentials(provider, email, token, secret));
     }
 
+    /** Remove a credential and persist the updated credentials to our preferences provider. */
+    public void removeCredentialAndPersist(@NonNull final String email) {
```

Comment:
There does need to be a way for other classes to call the removal of a user. It doesn't have to be this specific way or in some other way that addresses issue #344 . I personally think it should be in CredentialsManager and function in a similar fashion as this method. Until we have a replacement, I'm inclined to leave it as is.